content/news/2022-07-29-bevy-0.8/index.md

```diff
@@ -1208,3 +1208,408 @@ A huge thanks to the **X contributors** that made this release (and associated d
+~~- Only change the animation once in the `many_foxes` example (#5076)~~
+~~- Change panicking test to not run on global `TaskPool` (#4998)~~
+~~- Tweak the `transparency_3d` example (#4968)~~
+~~- Update `tests/how_to_test_systems.rs` to include using events (#4951)~~
+~~- Cleanups in diagnostics (#3871)~~
+~~- ECS benchmarks organization (#5189)~~
+~~- Describe new delegation strategy (#4562)~~
+~~- Public access for AnimationClip.duration (#4615)~~
+~~- Stop labeling PRs with Needs-Triage (#4686)~~
+~~- GitHub Actions: don't depend on patch versions (#4646)~~
+~~- Clean up advice on glob imports in style guide (#4644)~~
+~~- move system_param fetch struct into anonymous scope to avoid name collisions (#4100)~~
+~~- Remind users to initialize their systems before running them (#3947)~~
+~~- Change `gamepad.rs` tuples to normal structs (#4519)~~
+~~- Tidy up PluginGroupBuilder by moving Plugin index retrieval to it's own function (#4446)~~
+~~- Rename `transparent_phase` to `opaque_phase` in `wireframe.rs` (#4639)~~
+~~- Make AnimationClip::duration return value instead of reference (#4617)~~
+~~- Derive thiserror::Error for HexColorError (again) (#4847)~~
+~~- shader examples wording coherence (#4810)~~
+~~- remove unneeded msaa explicit addition from examples (#4830)~~
+~~- Make bevy_app's optional bevy_reflect dependency actually optional (#4846)~~
+~~- Rename Color::as_hlsa_f32 to Color::as_hsla_f32 (#4827)~~
+~~- (ManevilleF/staging) Remove markdown dead link check (#4839)~~
+~~- Reduce code duplication by using QueryIterationCursor in QueryIter (#4733)~~
+~~- Change path to zld on MacOS fast build example (#4778)~~
+~~- Doc/module style doc blocks for examples (#4438)~~
+~~- Remove strong language from CONTRIBUTING.md (#4755)~~
+~~- Update commented vsync code in example to use present_mode (#4926)~~
+~~- Do not bundle the assets from wasm example in the crate (#4895)~~
+~~- Derive default on ReportExecutionOrderAmbiguities (#4873)~~
+~~- Update "C-Bug" label and url in CONTRIBUTING.md (#4880)~~
+~~- Clippy improvements (#4665)~~
+~~- Make bug template more beginner-friendly (#4652)~~
+~~- Cargo `--timings` option has been stabilized. Update profiling.md. (#4850)~~
+~~- Remove redundant query parameters (#4945)~~
+~~- Recommend posting new plugins in #crates discord channel (#4956)~~
+~~- Let the project page support GitHub's new ability to display open source licenses (#4966)~~
+~~- Enable single-commit option for doc deployment (#5138)~~
+~~- Remove double blank line from component docs (#5102)~~
+~~- Default image sampler doc fix (#5047)~~
+~~- Remove spirv-reflect from readme (#5192)~~
+~~- remove an unused import in release (#5320)~~
+~~- remove component and resource suffixes from reflect structs (#5219)~~
+~~- Allow unicode license (#5337)~~
+~~- Remove some unused dependencies (#4544)~~
+~~- Misc dependency improvements (#4545)~~
+~~- Decouple some dependencies (#3886)~~
+~~- Update ndk-glue requirement from 0.5 to 0.6 (#3624)~~
```

Comment:
IMO dependency bumps should be listed, but likely in a separate section (or all together) to reduce noise.

Reply:
Sounds good to me, but I'll leave that for @cart.